tests/environment.bats

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bats
 
-load '/usr/local/lib/bats/load.bash'
+load "$BATS_PLUGIN_PATH/load.bash"
```

Comment:
This is not directly related to the PR itself, but was done because `make tests` started [failing on this](https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/401#01878078-eff5-4a1d-a188-5187577048dd/120-148), making the PR go red.
I figured that the plugin-tester docker image might have been updated recently and broke things, so I took a look at [the plugin-tester readme](https://github.com/buildkite-plugins/buildkite-plugin-tester#usage) and noticed they were recommending to use this env var here, and that indeed fixed the CI issue.